MobileCenter/MobileCenterTests/MSMobileCenterTests.m

```diff
@@ -13,6 +14,9 @@
 #endif
 #import "MSMockUserDefaults.h"
 #import "MSTestFrameworks.h"
+#import "MSCustomProperties.h"
```

Comment:
The import order should be alphabetical but let's keep this and do the clean up later.